.jenkins/library/vars/globalvars.groovy

```diff
@@ -11,18 +11,28 @@ import groovy.transform.Field
 @Field CTEST_TIMEOUT_SECONDS = 480
 @Field GLOBAL_ERROR = null
 @Field AGENTS_LABELS = [
-    "acc-ubuntu-20.04":         env.UBUNTU_2004_CUSTOM_LABEL ?: "ACC-2004",
-    "acc-v3-ubuntu-20.04":      env.UBUNTU_2004_ICX_CUSTOM_LABEL ?: "ACC-v3-2004",
-    "acc-ubuntu-18.04":         env.UBUNTU_1804_CUSTOM_LABEL ?: "ACC-1804",
-    "acc-v3-ubuntu-18.04":      env.UBUNTU_1804_ICX_CUSTOM_LABEL ?: "ACC-v3-1804",
-    "ubuntu-nonsgx":            env.UBUNTU_NONSGX_CUSTOM_LABEL ?: "nonSGX-ubuntu-2004",
-    "ubuntu-nonsgx-20.04":      env.UBUNTU_NONSGX_CUSTOM_LABEL ?: "nonSGX-ubuntu-2004",
-    "ubuntu-nonsgx-18.04":      env.UBUNTU_NONSGX_CUSTOM_LABEL ?: "nonSGX-ubuntu-1804",
-    "windows-nonsgx":           env.WINDOWS_NONSGX_CUSTOM_LABEL ?: "nonSGX-Windows",
-    "acc-ubuntu-20.04-vanilla": env.UBUNTU_VANILLA_2004_CUSTOM_LABEL ?: "vanilla-ubuntu-2004",
-    "acc-ubuntu-18.04-vanilla": env.UBUNTU_VANILLA_1804_CUSTOM_LABEL ?: "vanilla-ubuntu-1804",
-    "acc-win2019-dcap":         env.WINDOWS_2019_DCAP_CUSTOM_LABEL ?: "SGXFLC-Windows-2019-DCAP",
-    "acc-v3-win2019-dcap":      env.WINDOWS_2019_DCAP_ICX_CUSTOM_LABEL ?: "ACC-v3-SGXFLC-Windows-2019-DCAP",
-    "shared":                   "Jenkins-Shared-DC2"
+    // ACC VMs
+    "acc-ubuntu-20.04":            env.UBUNTU_2004_CUSTOM_LABEL ?: "ACC-2004",
+    "acc-v3-ubuntu-20.04":         env.UBUNTU_2004_ICX_CUSTOM_LABEL ?: "ACC-v3-2004",
+    "acc-ubuntu-18.04":            env.UBUNTU_1804_CUSTOM_LABEL ?: "ACC-1804",
+    "acc-v3-ubuntu-18.04":         env.UBUNTU_1804_ICX_CUSTOM_LABEL ?: "ACC-v3-1804",
+    "acc-win2019-dcap":            env.WINDOWS_2019_DCAP_CUSTOM_LABEL ?: "SGXFLC-Windows-2019-DCAP",
+    "acc-v3-win2019-dcap":         env.WINDOWS_2019_DCAP_ICX_CUSTOM_LABEL ?: "ACC-v3-SGXFLC-Windows-2019-DCAP",
+    // Non SGX VMs
+    "ubuntu-nonsgx":               env.UBUNTU_NONSGX_CUSTOM_LABEL ?: "nonSGX-ubuntu-2004",
+    "ubuntu-nonsgx-20.04":         env.UBUNTU_NONSGX_CUSTOM_LABEL ?: "nonSGX-ubuntu-2004",
+    "ubuntu-nonsgx-18.04":         env.UBUNTU_NONSGX_CUSTOM_LABEL ?: "nonSGX-ubuntu-1804",
+    "windows-nonsgx":              env.WINDOWS_NONSGX_CUSTOM_LABEL ?: "nonSGX-Windows",
+    // Plain VMs
+    "acc-ubuntu-20.04-vanilla":    env.UBUNTU_VANILLA_2004_CUSTOM_LABEL ?: "vanilla-ubuntu-2004",
+    "acc-ubuntu-18.04-vanilla":    env.UBUNTU_VANILLA_1804_CUSTOM_LABEL ?: "vanilla-ubuntu-1804",
+    // US-specific ACC VMs
```

Comment:
what does US refer to?

Reply:
United States. These are useful for access control and specific tests as typically these agents run a little bit faster